com.github.alexkdeveloper.notepad.json

```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "com.github.alexkdeveloper.notepad",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.github.alexkdeveloper.notepad",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
```

Comment:
don't you need DRI too?

Reply:
What for? The app is already working well)